backend/corpora/lambdas/api/v1/dataset.py

```diff
@@ -50,6 +49,8 @@ def get_status(dataset_uuid: str, user: str):
     dataset = Dataset.get(dataset_uuid)
     if not Collection.if_owner(dataset.collection.id, dataset.collection.visibility, user):
         raise ForbiddenHTTPException()
+    if dataset.tombstone:
```

Comment:
does this work for legacy datasets that don't have tombstone column set

Reply:
I assumed they all got set in the migration script? I can wrap it in a try except or switch to .get

Reply:
A simple test could verify this.